src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age ? <p className="Person__age">I am {person.age}</p> : null}
+    <p className="Person__partner">
+      {person.isMarried === true ? (
+        <p>
+          {person.partnerName} is my {person.sex === 'f' ? 'husband' : 'wife'}
+        </p>
+      ) : (
+        <p>I am not married</p>
+      )}
```

Comment:
decide before jsx what text will be rendered and just render in jsx. something like that

```
const personPartnerType = sex === 'm' ? 'wife' : 'husband';
const personPartner = isMarried ? `${partnerName} is my ${personPartnerType}` : 'I am not married'
```